### ISSUE TITLE
The compiled APK crashes with error on _imaging.so: not found

### ISSUE THREAD

**Issue by GijsTimmers** [kivy/python-for-android#335]
I want to compile a Python program. I'm using the following steps to do so:

1. `git clone git://github.com/kivy/python-for-android`
2. `cd python-for-android`
3. `./distribute.sh -m "openssl pil kivy mechanize pygments sqlite3" -f`
4. `cd dist/default`
5. `./build.py --package org.me.myname --name "proof-of-concept" --version 1.3 --dir /home/me/Scripts/proof-of-concept/app-development --orientation portrait --window debug`
6. `adb install bin/proof-of-concept-1.3-debug.apk`

The app will then install successfully, but it crashes immediately after the `presplash`, with this error:

    03-12 12:27:52.972: E/dalvikvm(9496): dlopen("/data/data/org.me.myname/files/lib/python2.7/lib-dynload/_imaging.so") failed: dlopen failed: library "/data/data/org.me.myname/files/lib/python2.7/lib-dynload/_imaging.so" not found

I don't know what this could be, I've searched online and read posts telling me to include `pil`, `sqlite3`, and `openssl`, but even when doing so, this error persists.

Could you help me?

**Comment by FeralBytes** [comment]
I have two recommendations for you. 
1) Use Buildozer and see if the error persists when you are using Buildozer.
2) We would want to see a full adb logcat log before even trying to look into it, but please don't post it as a comment, instead make a gist and then link to that gist in a comment here.
Thanks.

**Comment by GijsTimmers** [comment]
Hi, thank you for your quick answer. I tried using Buildozer, but I've been having another error there (`[ant, debug]`), so because of that, I moved to `python-for-android`.

I'll post the logcat as soon as this happens again, but since I have rewritten the code, it hasn't been happening again. I'm almost sure it has something to do with the imports in Python. I'll be closing this issue until it happens again, and when it does, I'll post a full logcat.
